Source/Core/Resource.js

```diff
@@ -1570,7 +1570,7 @@ define([
      * using XMLHttpRequest, which means that in order to make requests to another origin,
      * the server must have Cross-Origin Resource Sharing (CORS) headers enabled.
      *
-     * @param {Object} data Data that is posted with the resource.
+     * @param {*} data Data that is posted with the resource.
```

Comment:
I _believe_ the data you post must be an `Object` or a `string`.

Reply:
Yeah, it should be `Object`